wp-content/themes/workingnyc/src/js/modules/programs-archive.js

```diff
@@ -187,7 +227,152 @@ export default {
           }
         }
       })(_this);
-    }
+    },
+
+    process: function(data, query, headers) {
```

Comment:
Can you add comments for each of the functions you added indicating:
* What the function does
* If it's overriding a function from the archive.vue package, what you have changed here?